src/pymovements/dataset/dataset.py

```
@@ -139,13 +139,6 @@ def scan(self) -> Dataset:
         -------
         Dataset
             Returns self, useful for method cascading.
-
-        Raises
```

I'm not sure if I agree to remove the raises section. is pydoclint complaining?

---

pydoclint complains that the function does not have a raise in it, it just stupidly checks for raises within scope of function :-( -- ignore?

---

yes, let's ignore all errors from Raises and Attributes sections for this PR

I hope there's some code like DXXX for this

---

there's no error code, see my latest main comment